src/components/pages/coach/ReportList/index.jsx

```diff
@@ -75,42 +98,39 @@ class ReportList extends React.Component{
     componentDidMount = () => {
         // Set page title
         document.title = "Beautyreport List";
-        
-        if(this.props.location){
-            if(this.props.location.state){
-                if(this.props.location.state.user.id){
+        if(this.props.match){
+            if(this.props.match.params){
+                if(this.props.match.params.uid){
                     this.setState({
-                        user: this.props.location.state.user,
-                        loading: true
-                    }, () => this.fetchReports(this.props.location.state.user.id));
+                        uid: this.props.match.params.uid,
+                        loading: true,
+                    }, () => this.fetchReports(this.props.match.params.uid));
                 }
             }
         }
     }
 
-    componentWillUnmount = () => {
-        //clearInterval(this.interval);
-    }
-
     fetchReports = (uid) => {
         //this.interval = setInterval(() => this.fetchAllReports(uid), 5000);
         // Fetch data required for creating a report
         this.fetchAllReports(uid);
     }
 
     fetchAllReports = (uid) => {;
-        this.props.client.query({
+        clientNoCache.query({
         query: GET_REPORTS,
         variables: { "id": uid, "token": localStorage.getItem("fprint") }
```

Comment:
"fprint" doesn't seem to be the best naming for an JWT in local storage.
I would prefer to name it anonymousJWT or something along those lines,
cause it's the JWT of the anonymous user.

Reply:
I have to agree on that one. They are a remenant of darker times. 🌋